.rubocop.yml

```diff
@@ -42,7 +42,7 @@ Metrics/ModuleLength:
   Max: 350
 
 Metrics/CyclomaticComplexity:
-  Max: 10
+  Max: 15
```

Comment:
Looks like the new algorithm was detecting a bunch of new warnings with 10 so upped to 15 to aim for least amount of changes. Keen to drop these as I feel like they just slow us down and don't actually add much values as we usually disable them and never review this.

Reply:
Yeah I have mixed feelings about this, I find that codebases where we're strict about following this tend to be easier to grok and change over time, but at this point there are so many exceptions that it kind of defeats the point of having the rule.

Reply:
hm yeah true, should we be more strict and not introducing any new disables and getting rid of any we encounter? Up for that.

Reply:
Yeah if you're up for it let's do it! 🎉